<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow {
background-color: rgba(51, 51, 51);v

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="VideoFrameDisplay" name="widget_video" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="VideoSlider" name="video_slider">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>15</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
border: 0px solid #bbb;
}

QSlider::sub-page:horizontal {
background-color: #ff9900;
border-bottom-left-radius: 2px; 
border-top-left-radius: 2px; 

border-radius: 2px; 
margin-top:5px;
margin-bottom:5px;
}

QSlider::add-page:horizontal {
background-color: rgba(228, 228, 228, 50);v
border: 0px solid #777;
border-radius: 2px;
margin-top:5px;
margin-bottom:5px;
}
/*
QSlider::handle:horizontal {
background: rgb(255,153,102); 
border: 1px solid rgb(255,153,102); 
width: 14px;
height:10px;
border-radius: 7px;
margin-top:2px;
margin-bottom:2px;
}

QSlider::handle:horizontal:hover {
background: rgb(255,128,6); 
border: 1px solid rgba(102,102,102,102);
border-radius: 7px;
}*/

QSlider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 2px;
}
</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="ratio">
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: white
}</string>
      </property>
      <property name="text">
       <string>pts progress</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="start">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Play a video</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stop_video">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>pause/resume</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>799</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1132</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoFrameDisplay</class>
   <extends>QWidget</extends>
   <header>videoframedisplay.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VideoSlider</class>
   <extends>QSlider</extends>
   <header>VideoSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
